main.py

```diff
@@ -14,6 +15,25 @@ def tweet(message: str):
 
     api = tweepy.API(auth)
     api.update_status(message)
+    logger.info("The gazette was posted on twitter!")
+
+
+def refresh_maria_quiteria_api_token():
+    headers = {
+        "content_Type": "application/json"
+    }
+    url = os.getenv("MARIA_QUITERIA_API_URL_TOKEN")
```

Comment:
Uma boa prática é ter o HOST como variável e adicionar os endpoints no código. COmo não tínhamos acesso a mais de um endpoint, antes era aceitável. Agora, ao invés de uma nova variável pra cada endpoint, precisamos ter apenas uma variável para o host e criar a url onde necessário.
```suggestion
    url = f"{os.getenv('MARIA_QUITERIA_API_HOST')}/token/refresh/"
```
Lembrando que o host não deve ter uma barra ao final.